packages/synapse-react-client/src/synapse-queries/entity/useEntityBundle.ts

```diff
@@ -6,26 +6,60 @@ import {
   EntityBundleRequest,
 } from '@sage-bionetworks/synapse-types'
 import {
+  skipToken,
   useQuery,
   UseQueryOptions,
   useSuspenseQuery,
+  UseSuspenseQueryOptions,
 } from '@tanstack/react-query'
 
 export function useGetEntityBundleQueryOptions<
   T extends EntityBundleRequest = typeof ALL_ENTITY_BUNDLE_FIELDS,
+  TSelect = EntityBundle<T>,
 >(
-  entityId: string,
+  entityId?: string,
   version?: number,
   bundleRequest: T = ALL_ENTITY_BUNDLE_FIELDS as T,
-) {
+  select?: (data: EntityBundle<T>) => TSelect,
+): UseQueryOptions<EntityBundle<T>, SynapseClientError, TSelect> {
   const { accessToken, keyFactory } = useSynapseContext()
   return {
+    select,
     queryKey: keyFactory.getEntityBundleQueryKey(
       entityId,
       version,
       bundleRequest,
     ),
+    queryFn: entityId
+      ? () =>
+          SynapseClient.getEntityBundleV2<T>(
+            entityId,
+            bundleRequest,
+            version,
+            accessToken,
+          )
+      : skipToken,
+  }
+}
 
+export function useGetEntityBundleSuspenseQueryOptions<
```

Comment:
We can't use `skipToken` in suspense queries, so I had to create a separate function to ensure we return the correct type when we get the `UseSuspenseQueryOptions`